src/OverflowVisibleContainer/OverflowVisibleContainer.tsx

```diff
@@ -41,7 +41,8 @@ export const OverflowVisibleContainer = React.forwardRef<Ref, Props>(
 				<div
 					ref={ref}
 					className={cn("kit-overflow-visiblecontainer", className, {
-						"kit-overflow-visiblecontainer_fixed": isFixed
+						"kit-overflow-visiblecontainer_fixed": isFixed,
+						"kit-overflow-visiblecontainer_open": hidden
```

Comment:
```suggestion
						"kit-overflow-visiblecontainer_open": !hidden
```
Может вообще лучше сделать модификатор `_hidden` в котором будет `dispaly: none`?